en/backup-restore-cr.md

```diff
@@ -258,6 +258,32 @@ This section introduces the fields in the `Backup` CR.
 * `.spec.local.volume`: the persistent volume configuration.
 * `.spec.local.volumeMount`: the persistent volume mount configuration.
 
+## CompactBackup CR fields
+
+To compact log backup data into structured SST files, users can create a custom CompactBackup CR object to describe a backup task. Below is a detailed explanation of the fields in the CompactBackup CR:
```

Comment:
add a sentence saying "From TiDB version 9.0.0 and above, you can compact log backup to accelerate PITR restore。"